src/glb-healthcheck/main.go

```diff
@@ -77,16 +82,39 @@ Options:
 		return
 	}
 
-	// the check manager will run the HC loop and manage most of the HC part of the work
-	ctx.checkManager = NewHealthCheckManager(HealthCheckTimeout, HealthCheckInterval)
-
 	// load up the forwarding table and register all the check targets
 	err = ctx.LoadForwardingTable()
 	if err != nil {
 		ctx.logContext.Fatalf("Could not load initial forwarding table: %v", err)
 		return
 	}
 
+	if ctx.forwardingTableConfig.HealthcheckGlobalCfg != nil &&
+		ctx.forwardingTableConfig.HealthcheckGlobalCfg.TimeoutMilliSec != 0 {
+		HealthCheckTimeout = ctx.forwardingTableConfig.HealthcheckGlobalCfg.TimeoutMilliSec * 1000000
```

Comment:
I believe the (slightly weird) Go idiom is to multiply the value in a unit by `time.<Unit>` to get it into a `time.Duration`, so I think this is the same:
```suggestion
		HealthCheckTimeout = ctx.forwardingTableConfig.HealthcheckGlobalCfg.TimeoutMilliSec * time.Millisecond
```